test/unit/utils-defs-test.js

```diff
@@ -42,7 +42,6 @@ describe('utils definitions', function() {
         expect(typeof utils.css, 'css function is defined').to.equal('function');
         expect(typeof utils.style, 'style function is defined').to.equal('function');
         expect(typeof utils.clearCss, 'clearCss function is defined').to.equal('function');
-        expect(typeof utils.transform, 'transform function is defined').to.equal('function');
```

Comment:
Who said you could remove `transform` from utils? This test is here to verify the interface of our API which you've changed.